test/cri-containerd/disable_vpmem_test.go

```diff
@@ -0,0 +1,107 @@
+//go:build functional
+// +build functional
+
+package cri_containerd
+
+import (
+	"context"
+	"crypto/rand"
+	"encoding/base64"
+	"fmt"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/Microsoft/hcsshim/pkg/annotations"
+	runtime "k8s.io/cri-api/pkg/apis/runtime/v1alpha2"
+)
+
+// Use unique names for pods & containers so that if we run this test multiple times in parallel we don't
+// get failures due to same pod/container names.
+func uniqueRef() string {
```

Comment:
Is this just for safety or do we really need it? We wouldn't expect to run this test multiple times in parallel, I don't think. Probably a lot of other tests would have similar problems if we started doing that.

Reply:
I needed it for this test because I run this test several times (with `test.parallel=3`) because that's when we hit the original issue of disk showing up on a different SCSI controller.